build/pipelines/templates/run-staticAnalysis.yaml

```diff
@@ -9,7 +9,7 @@ steps:
     displayName: 'Install PSScriptAnalyzer'
 
   - powershell: |
-       $results = Invoke-ScriptAnalyzer -Path ./ –Recurse
+       $results = Invoke-ScriptAnalyzer -Settings ./PSScriptAnalyzerSettings.psd1 -Path ./ –Recurse
```

Comment:
Can you make a similar change to CONTRIBUTING.md as well?

https://github.com/microsoft/PowerShellForGitHub/blob/742b85324e94cfc1b3ba14edc8a450cccdc882fa/CONTRIBUTING.md#L152-L157

Reply:
Done. + another reference